### DISCUSSION TITLE
Hi Folks I m new here as a user of [Oct 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4240] [Discord Threads]
> 🙋‍♂️ **ence.prastama**  
#### ```  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1029992205068292136/error_log.png)

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Can you raise an issue with more details like    
####    
#### - Could you able to start nocodb ?    
#### - When it's happening ?  
#  
> 👨‍💻 **pranavxc**  
#### The configuration looks correct  
#  
> 🙋‍♂️ **ence.prastama**  
#### thank you <@682854952657616902> for your response. sure! I was trying to login to my github but forgot my pass and lost my recovery codes 😆 so will raise an issue after I'm able to login   
####    
#### for your questions:   
#### 1. yes I was able to start nocodb   
#### 2. so I am using Google Cloud Run with CPU on Request (so the instance will be inactivated once there's no request). My suspicion is that it's throwing error when it was idle but then there was request coming in. because after instance restart, it was working perfectly 🤔 do you have suggestions on how to enable debug log so maybe I could give you more information from the logs?  
#  
> 👨‍💻 **pranavxc**  
#### Thanks for sharing the details, it could be related to connection pool and on error we may need to close the connection properly. You can create an issue whenever possible and I will look into it .  
#  
> 🙋‍♂️ **ence.prastama**  
#### Hi <@682854952657616902> finally I can logged in into my github account, as promised, here is the issue I opened https://github.com/nocodb/nocodb/issues/4115   
####    
#### let me know if you have any inquiries  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1029992205206704169)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
